docs/releases.md

```diff
@@ -0,0 +1,48 @@
+# Release process
+
+The release process in broken up into 3 steps.
+
+1. Increment the version and add a new tag
+2. Modify the changelog
+3. Merge the release branch and create a "github release"
+
+## 1. Increment the version and add a new tag
+
+To update the version in package.json and create a tag run:
+
+```shell
+$ yarn version
+```
+
+The version number should be updated following [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200). The only change should be to package.json, and there should be a new tag on the commit prefixed with a "v".
+
+## 2. Modify the changelog
+
+`CHANGELOG.md` highlights changes that are relevant for consumers of the package. Our changelog follows [keepachangelog](https://keepachangelog.com/en/1.0.0/).
+
+### Steps to append to the changelog
+
+Immediately after a version commit & tag, run the following command to create a draft list of changes relating to the new version.
+
+```shell
+$ gulp changelog
+```
+
+This command will generate a partial changelog and output it to the console. Copy the text and paste it above the previous version in `CHANGELOG.md` and be sure to leave a blank line between versions. The output is git logs between the current new version and the previous released version. It will be important to edit and trim down that list so only the user facing changes are clear, concise and obvious.
```

Comment:
```suggestion
This command will generate a partial changelog and output it to the console. Copy the text and paste it above the previous version in `CHANGELOG.md` and be sure to leave a blank line between versions. The output is git logs between the current new version and the previous released version. It will be important to edit and trim down that list so that only the consumer-relevant changes are included.
```

I changed the term "user" to "consumer" to make it clear that we're talking about the project consuming the package rather than the end user interacting with components from the package.